wp-content/themes/oceanwp-child/functions.php

```diff
@@ -197,7 +305,7 @@ function add_related_news_to_initiative_pages() {
             $query = new WP_Query( $args ); 
 
             ?>
-            <p><b>Related News:</b></p>
+            <h4><b>Related News:</b></h4>
```

Comment:
Refrain from using `b` and add the correct style if needed.
Whenever you run into `b` again, please clear it.

Reply:
I can still see `<b>`. Can you please remove it? See my comment about [typography](https://github.com/dev-a-better-world/ggrc_website/pull/19/files/d6e27ebadc78f648dd0c77d35e082d47312d9529..4c17ad738fec5305fc57ead291204029a978a03a#r815269221) styles. The `boldness` should come from the css definitions in the css file.